### ISSUE TITLE
Chat Moderation Bot

### ISSUE THREAD

**Issue by Hordesworn** [WarzoneMC/Warzone#238]
# Chat Moderation Bot

> Issue Preview

Since this version of Warzone began racking up players day after day, the community has noticed a large number of chat rule-breakers. 

> Problem

When a chat offence occurs, players tend to ignore it and simply flame them for spamming rather than actually reporting them. In the scenario that a staff member is not able to review the report or punish the chat rule-breaker, fights break out and players have to deal with a very annoying individual.

> Solution

Which leads me to a solution. If a bot/plugin was implemented to deal with these types of rule-breakers, we wouldn't have this problem.
**Here's an example:**
_A player excessively using capital letters in their text, the bot warns them silently.
If the player would continue, the bot would mute them for the time that the staff members usually mute for._ 

Somehow, the bot would have to look up the history of the player and ultimately decide which punishment to bestow upon the player.

> Conclusion

The purpose of this bot is to maintain a positive community status in the game and make sure that the players do not have to deal with the negativity that players spread to each other through game chat.

_- Horde_

**Comment by jellz** [comment]
It's not a _bot_ as much as it is just a _plugin_.  
TL;DR You want Warzone to get a "ChatControl" plugin of sorts.

**Comment by Virtualizes** [comment]
Basically, yeah.

**Comment by MinecraftMan101** [comment]
Fights are funny tho

**Comment by jellz** [comment]
@BennyDoesTheStuff Mind providing an explanation as to why this was closed? Surely you don't think fights are funny lol ;P

**Comment by BennyDoesTheStuff** [comment]
"Chat moderation Bot" 
- This Github repo is for "TGM" so an issue for a bot wouldn't belong here. That would be something you DM an admin.
